src/app/routes/enrolment/view-requests/components/expiration-info/expiration-info.component.html

```diff
@@ -23,7 +23,7 @@
   <ng-container *ngIf="(!removeExpDate && defaultValidityPeriod) || expirationTime">
     <br />
     This role will now be issued with an expiration date of
-    <b>{{ expirationTime || defaultValidityPeriod | timeShift | defaultDate }} UTC</b>
+    <b>{{ expirationTime / 1000 || defaultValidityPeriod / 1000 | timeShift | defaultDate }} UTC</b>
```

Comment:
What do you think about changing timeShift pipe to accept miliseconds instead of seconds? Or we can add optional property which will inform how it should be treated: as seconds or miliseconds. Then this line would like like this:
<b>{{ expirationTime || defaultValidityPeriod | timeShift: 'miliseconds' | defaultDate }} UTC</b>

Reply:
You can leave it as it is right now. Anyway if we go with miliseconds everywhere, we should late adjust it to handle miliseconds instead of seconds :) no need to think about parametrization.

Reply:
sounds good!